ground/src/main/java/com/google/android/ground/ui/datacollection/DropAPinTaskViewModel.kt

```diff
@@ -41,5 +41,5 @@ constructor(resources: Resources, private val uuidGenerator: OfflineUuidGenerato
   }
 
   private fun createFeature(point: Point): Feature =
-    Feature(uuidGenerator.generateUuid(), Feature.Type.LOCATION_OF_INTEREST, point)
+    Feature(Feature.Tag(uuidGenerator.generateUuid(), Feature.Type.LOCATION_FEATURE), point)
```

Comment:
Why do map features have different IDs than the point LOIs they represent? In that case we'd need to maintain a mapping between the two, which should be unnecessary.

Reply:
Generally they don't -- this code is in the DropAPin task, so I think @shobhitagarwal1612 is generating a new feature that represents the pin, not an LOI. Likely this should have a different Feature Type to indicate as much. The code only made it into this PR due to the Feature LOI type name refactor.

Reply:
I see.. In that case shouldn't the pin ID be fixed? Definitely shouldn't be marked as an LOI if it's being used as a placeholder/cursor for the task UI.

Reply:
Yep, I updated the types for both this and polygon drawing in the latest commit